tests/src/test/scala/org/http4s/HeadersSpec.scala

```diff
@@ -40,15 +40,6 @@ class HeadersSpec extends Http4sSpec {
       headers.get(org.http4s.headers.Cookie).map(_.values.length) must beSome(2)
     }
 
-    "Find the headers with DefaultHeaderKey keys" in {
```

Comment:
I removed this test in my last commit as it uses two headers now properly modeled. @rossabaker Do you think that's ok?